stereotype/model.py

```diff
@@ -67,16 +70,19 @@ def to_primitive(self, role: Role = DEFAULT_ROLE):
                 result[to_primitive_name] = value
         return result
 
-    def serialize(self, role: Role = DEFAULT_ROLE):
-        """Creates raw data from this Model. The role can be used together with declare_roles to exclude fields."""
-        return self.to_primitive(role)
+    def serialize(self, role: Role = DEFAULT_ROLE, context=None):
+        """
+        Creates raw data from this Model. The role can be used together with declare_roles to exclude fields.
+        :param context: Optional value opaque to stereotype, useful for serializing custom field types.
```

Comment:
If documenting one parameter, it's probably best to document them all, can you do it? Both here and in `to_primitive`. I suggest this, together with a small change to the method description:

```
        """
        Creates raw data from this Model, creating a copy of the data it was initialized with.
        :param role: Can be used together with declare_roles to exclude fields for certain roles.
        :param context: Optional value opaque to stereotype, useful for serializing custom field types.
        """
```